Dockerfile.ubi

```diff
@@ -23,15 +23,17 @@ USER root
 WORKDIR /work
```

Comment:
The build seems to fail:
```
69.70 internal/pkg/daemon/bpfrecorder/bpfrecorder_linux_amd64.go:26:9: undefined: toStringInt8
69.70 internal/pkg/daemon/bpfrecorder/bpfrecorder_linux_amd64.go:31:9: undefined: toStringInt8
```

Reply:
Will follow-up with Pranita, thanks

Reply:
Added a PR for Pranita to use as a basis of the fix https://github.com/pranitaT/security-profiles-operator/pull/3